ambry-api/src/main/java/com/github/ambry/frontend/NamedBlobPath.java

```diff
@@ -89,6 +89,13 @@ public static NamedBlobPath parse(String path, Map<String, Object> args) throws
    * @return the {@link NamedBlobPath} that indicates the parsing result from blobUrl.
    * @throws RestServiceException on parsing errors.
    */
+
+  // account/container/key
+  // ->
```

Comment:
can u clean up some of the code before submitting for review ? this will allow us to focus on the code that is actually changed instead of spending time figuring out what to look for.